dist/functional-app.html

```diff
@@ -6,7 +6,7 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
   <link rel="stylesheet" type="text/css" href="application.css">
-  <script src="functional-app.js"></script>
+  <script src="../src/functional-app.js"></script>
```

Comment:
You aren't supposed to change anything in this file! The browser doesn't understand `src` it has to be compiled.